bower.json

```diff
@@ -0,0 +1,20 @@
+// Bower.json and .bowerrc together make sure to install angular and
+// angular-route into client/lib. If client/lib does not exist, bower
+// will create the directory.
+
+{
+  "name": "solo",
+  "version": "0.0.0",
+  "authors": [],
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "client/lib"
+  ],
+  "dependencies": {
+    "angular": "~1.2.18",
```

Comment:
Yeah let's update to use the latest version, that was an oversight on my
part. The image is just a stand in for the mic icon, and I think the long
numbers describe the vector path that draws the mic image.
On Mar 17, 2015 8:16 AM, "Brian Hsu" <notifications@github.com> wrote:

> In bower.json
> <https://github.com/plucky-oatmeal/plucky-oatmeal/pull/17#discussion_r26581576>
> :
>
> > +// angular-route into client/lib. If client/lib does not exist, bower
> > +// will create the directory.
> > +
> > +{
> > +  "name": "solo",
> > +  "version": "0.0.0",
> > +  "authors": [],
> > +  "license": "MIT",
> > +  "ignore": [
> > +    "**/.*",
> > +    "node_modules",
> > +    "bower_components",
> > +    "client/lib"
> > +  ],
> > +  "dependencies": {
> > +    "angular": "~1.2.18",
>
> Interesting. Should we all use the latest version of lib in our project
> (like the lib in package.json)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plucky-oatmeal/plucky-oatmeal/pull/17/files#r26581576>
> .
>